src/wilson/dom.cljs

```diff
@@ -78,6 +79,38 @@
   ([ks rows]
    (table ks rows {})))
 
+
+(defn sorted-table
+  ([ks rows opts]
+   (let [state (r/atom {:sort-key (first ks)
+                        :sort-order :asc})
+         state-deref @state
+         sort-fns (or (:sort-fns opts)
+                      {:default (fn [k rows]
+                                  (if (= (:sort-order state-deref) :asc)
```

Comment:
This appears to use the state-deref when the component is instantiated. I guess that's what you mean?